mautic/mautic
# Max entity lock time

| Q  | A
| --- | ---
| Bug fix? | N
| New feature? | Y
| Related user documentation PR URL | https://github.com/mautic/documentation/pull/173
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/926
| BC breaks? | N
| Deprecations? | N

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Free automatically entity after a max lock time period.

#### Steps to test this PR:
1. Configure max entity lock time in configuration panel (120 sec to test for example), in system settings, Miscellaneous pannel
2. Create an email or an other item, save it, and go to modify page
3. in an other browser, with an other user, try to access to this item.
4. item should be lock 120sec after modification lock, and automatically free after 120sec.
